plugins/scaffolder-backend/sample-templates/create-react-app/skeleton/catalog-info.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   description: {{cookiecutter.description | jsonify}}
   annotations:
     github.com/project-slug: {{cookiecutter.destination.owner + "/" + cookiecutter.destination.repo}}
-    backstage.io/techdocs-ref: url:{{cookiecutter.destination.host + "/" + cookiecutter.destination.owner + "/" + cookiecutter.destination.repo}}
+    backstage.io/techdocs-ref: url:{{"https://" + cookiecutter.destination.host + "/" + cookiecutter.destination.owner + "/" + cookiecutter.destination.repo}}
```

Comment:
This is pretty unfortunate.

I wonder how we can make this nicer in the long run. 

Maybe breaking out the `github` action into a `createRepo` first, so that it would return the `fullUrl` from that action and do the templating after?

Or maybe a processor in the catalog. Anyways, don't think it for now at least.